### ISSUE TITLE
Feature in "View PDF" for drawing tool

### ISSUE THREAD

**Issue by friishon** [mozilla/pdf.js#18287]
Hi, can you add a rubber for delete the writings?
But instead of deleting all, just a part (similar to Samsung notes)

Thanks

**Comment by Snuffleupagus** [comment]
*For future reference:* Please remember to provide a more descriptive title since it's not really possible to understand what the issue is about from the title alone.
Furthermore, please actually fill out the ISSUE_TEMPLATE *properly* (rather than ignoring/deleting it) such that all relevant information is available.

**Comment by timvandermeij** [comment]
Yes, please provide all requested information and elaborate a bit more on what kind of feature you mean, because from the existing text I can't infer what kind of feature you're specifically asking for. Do you perhaps have screenshots of what Samsung Notes apparently offers?

**Comment by friishon** [comment]
Sorry

I want to add to the drawing button another option like a rubber for deleting the writing notes if i commit a mistake

 
![SmartSelect_20240619_171943_Samsung Notes_1](https://github.com/mozilla/pdf.js/assets/19889173/d68d538c-791e-43e3-a4b6-826a896b5dfd)

As you can see in the image, the re are 3 options 

1 -> pen
2 -> highlighter
3 -> rubber

for examen i draw something in a normal note
![Screenshot_20240619_173343_Samsung Notes_1](https://github.com/mozilla/pdf.js/assets/19889173/f6330ad7-be4b-4e9f-af4f-7ba2a56dc314)

and if i choose the rubber, i can delete only one part
![Screenshot_20240619_173457_Samsung Notes_1](https://github.com/mozilla/pdf.js/assets/19889173/fc704c4a-95d2-40bb-b501-9adf473b65db)

sorry because i did with my phone and maybe is not the best option

**Comment by lorimode** [comment]
That would indeed be a nice little extra to add 😉...

**Comment by marco-c** [comment]
We agree it would be nice, but unfortunately we'd first need to have a UX design to implement.

**Comment by lorimode** [comment]
A UX design? That's to say ?

**Comment by marco-c** [comment]
A specification of what it should look like graphically and how it should behave with respect of other editing tools.

**Comment by lorimode** [comment]
For example, something like this:

<img width="1920" height="919" alt="Image" src="https://github.com/user-attachments/assets/6a86c6ae-9ec5-4995-8358-3aab8399bc6f" />

<img width="1920" height="919" alt="Image" src="https://github.com/user-attachments/assets/c1a7e2e4-f7f2-47a3-aa31-6fc99bcc7b20" />

The eraser interacts with freehand/drawing type elements:

<img width="1920" height="919" alt="Image" src="https://github.com/user-attachments/assets/876beaf6-1d07-4869-8a1c-d12945671f6c" />

Then, we would also need to know if the eraser also affects other elements (free highlighting in particular)...

This is a quick little example (be made in montage), tell me what you think?
